sdk/identity/identity-cache-persistence/CHANGELOG.md

```diff
@@ -1,16 +1,11 @@
 # Release History
 
-## 1.0.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.1.0 (2024-03-05)
```

Comment:
1.1.0 or 2.0.0 both felt appropriate, although 1.1.0 is most consistent with our [support policy](https://azure.github.io/azure-sdk/policies_support.html#azure-sdk-dependencies)

Reply:
Then i think 1.1.0 should be ok